src/main/java/hudson/plugins/selenium/HubLauncher.java

```diff
@@ -37,21 +37,21 @@ public HubLauncher(int port, String[] args, Level logLevel) {
 
     public Void call() {
         try {
-            Logger LOG = Logger.getLogger(HubLauncher.class.getName());
+            Logger log = Logger.getLogger(HubLauncher.class.getName());
```

Comment:
@Mobrockers Yes sure, because here it was non-static final instance so this is the correct naming convention for it.